modernizer-annotation-processor/src/main/java/org/gaul/annotation_processsor/ModernizerAnnotationProcessor.java

```diff
@@ -148,4 +199,71 @@ private String getFullClassName(Element classElement) {
         }
         return className.toString() + classElement.getSimpleName();
     }
+
+    private String getMethodIdentifierString(Element methodElement) {
+        List<? extends TypeMirror> methodParams =
+            ((ExecutableType) methodElement.asType()).getParameterTypes();
+        String fullClassPattern =
+            getClassHeader(methodElement.getEnclosingElement());
```

Comment:
Does this handle methods defined inside of anonymous classes?

Reply:
No. Not sure if there's a way to retrieve methods from an anonymous class in the processor. 

https://stackoverflow.com/questions/35802384/retrieve-elements-within-anonymous-class-using-annotation-processing

Reply:
That stackoverflow makes it sound like our processor won't get invoked at all for anonymous class methods, which seems fine. My concern was that it would get invoked for one of these methods, but then we would generate an invalid class pattern